src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterCheckFormat.bash

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+: '
+SuppressWithNearbyTextFilter
```

Comment:
We do not support any comments other than java multiline.
But nothing stops us to use such comment in text file.
We do not care about file to be valid sh or css. For us all of them some text files.

Reply:
I know checkstyle is not picking up this , but I left it there just to keep consistency and make it easy for the reader know what the configuration for the file is.

Reply:
Alright, done.